styles/style.css

```diff
@@ -0,0 +1,93 @@
+body {
+background-color: white;
+}
+
+#main {
+height: 1700px;
+width: 1000px;
+margin: auto auto;
+border-style: inherit;
+border: 1px solid black;
+clear: left;
```

Comment:
@Vovan707 Please add leading 2 space indents for all css code inside `{ }` block.